docs/api_v2/api_v2_reference.apib

```diff
@@ -2918,238 +3123,413 @@ The user account is identified by the provided authentication token.
 
 ## Build (object)
 - id:               `HG7ML7M8z78YcAPEB`        (string, required)
-- actId:            `johndoe~my-actor`         (string, required)
+- actId:            `janedoe~my-actor`         (string, required)
 - userId:           `klmdEpoiojmdEMlk3`        (string, required)
-- startedAt:        `2015-11-30T07:34:24.202Z` (string, required)
-- finishedAt:       `2015-12-12T09:30:12.202Z` (string, nullable)
+- startedAt:        `2019-11-30T07:34:24.202Z` (string, required)
+- finishedAt:       `2019-12-12T09:30:12.202Z` (string, nullable)
 - status:           `SUCCEEDED`                (string, required)
 - meta                                         (object, required)
-    - source:        `WEB`                     (string, required)
+    - origin:        `WEB`                     (string, required)
     - clientIp:      `172.234.12.34`           (string, required)
     - userAgent:     `Mozilla/5.0 (iPad)`      (string, required)
 - stats                                        (object, nullable)
```

Comment:
Ouch, we should remove `runTimeSecs`, `readyTimeSecs `, `initTotalSecs` and `emfileError` as those are internal and could confuse users. What do you think @jancurn ?

Reply:
Yeah defo, no need to share these

Reply:
do you want to remove them just from docs or also from API? I think the idea was to have both fully consistent